### DISCUSSION TITLE
Attempting to publish npm package when nothing has been built.

### DISCUSSION THREAD

**Discussion by jchoi926** [semantic-release/semantic-release#1969] [Q&A] [answered]
Im using a combination of nx to manage a monorepo and semantic-release to manage releases. The issue that Im facing is when an app in the monorepo has a build/release and the npm package does not, the publishing to our private npm registry still occurs.
Is there some way to configure the publishing to occur only if the npm package has been built?

**Comment by travi** [comment, marked as answer]
semantic-release does not support monorepos and expects all commits to be for the package that the repo is for. there are some unofficial extensions of semantic-release that have taken steps to understand when changes apply to packages inside of a monorepo, but that is outside of the supported use-cases that are officially supported
